runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/BenefitsSection/BenefitsCarusel/BenefitsCarusel.module.scss

```diff
@@ -0,0 +1,141 @@
+@import "#src-landing/scss/main.scss";
+
+$BEFORE-X-OFFSET: 20px;
+$BEFORE-Y-OFFSET: 40px;
+
+$CARD-BORDER-RADIUS: 20px 20px 150px 20px;
+$CARD-TEXT-MAX-WIDTH: 250px;
+$COUNTER-MIN-WIDTH: 120px;
+
+$CARD-CONTAINER-ICON-QUERY: 325px;
+
+.root {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 100%;
+  isolation: isolate;
+}
+
+.buttonWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-right: $BEFORE-X-OFFSET;
+  position: relative;
+}
+
+.contentWrapper {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  width: 532px;
+  height: 100%;
+  min-height: 500px;
+  margin: 0 $margin-2;
+  // padding to offset border radius for correct word wrapping
+  padding: 90px 30px 70px 20px;
+  border-radius: $CARD-BORDER-RADIUS;
+  background-color: $palette-background-primary;
+  box-shadow: 0px 16px 26px 2px #0000001f;

```

Comment:
hmm, are box shadows repetitive? if so then imo we should move them to global styles and use predefined color